### PR TITLE
Dian.garkov issue #146

### PR DESCRIPTION
**Pull request by diangarkov**
I updated the readme file and added automated tests for all requirements. I included the tests in the oneClickInstaller but if you find that they sould be separately executed, it is also okay - but in my opinion, we must have some kind of automated tests.

### CODE REVIEW

File: oneClickInstaller.sh [hunk 1]
```diff
@@ -1,4 +1,6 @@
 #!/usr/bin/env bash
+#check requisites
+sh ./checkPreRequisites.sh
```
Comment (muralikrishna8) [inline oneClickInstaller.sh]: looks like you forgot to include `checkPreRequisites.sh` file.

File: README.md [hunk 2]
```diff
@@ -1,29 +1,55 @@
...
 2. Install couchdb 2.0
-3. Install couchdb lucene 2.0
+3. Create an admin user in couchdb with whom you can create, delete and change databases manually on the localhost:5984/_utils/# <br />
+If your admin user in couchdb is not with the name 'admin' and password 'admin', change this data in the file \" instal.config \" from this very project.
+4. Install couchdb lucene 2.0
+5. Install gulp
+
+
+## Test Requisites:
+1. write 'npm --version' in the console and it shall return the number of the certain installed npm version
```
Comment (muralikrishna8) [inline README.md]:
```suggestion
1. write `npm --version` in the console and it shall return the number of the certain installed npm version
```

File: README.md [hunk 3]
```diff
@@ -1,29 +1,55 @@
 # Makenews
 
 ## Pre Requisites:
-1. Install node(version >= 4.0)   npm
+1. Install node(version >= 4.0) npm
```
Comment (muralikrishna8) [inline README.md]:
```suggestion
1. Install node(version >= 8.0) npm
```

File: README.md [hunk 4]
```diff
@@ -1,29 +1,55 @@
...
+
+## Test Requisites:
+1. write 'npm --version' in the console and it shall return the number of the certain installed npm version
+2. write 'localhost:5984' on your browser and there you should see the couchdb version and some additional information about couchdb.
+3. check whether you have configurate correctly the couchdb-lucene. For instance,
+you can try querying a database(if you created database in couchdb and you have installed curl) like this:
+```
+	curl http://"your_database_ip":5984/database_name/_fti/design_doc/view_name?q=Query
+```
+4. write 'gulp --version' in the console and it shall return the number of the certain installed gulp version(local and cli)
```
Comment (muralikrishna8) [inline README.md]:
```suggestion
4. write `gulp --version` in the console and it shall return the number of the certain installed gulp version(local and cli)
```

File: README.md [hunk 5]
```diff
@@ -1,29 +1,55 @@
...
+5. write 'git --version' in the console and it shall return the number of the certain installed git version
+
+## Recommended software:
+1. Install git- for cloning the git repository
```
Comment (muralikrishna8) [inline README.md]: I think `git` as a pre-requisite is not necessary.

File: README.md [hunk 6]
```diff
@@ -1,29 +1,55 @@
 # Makenews
 
 ## Pre Requisites:
-1. Install node(version >= 4.0)   npm
+1. Install node(version >= 4.0) npm
 2. Install couchdb 2.0
-3. Install couchdb lucene 2.0
+3. Create an admin user in couchdb with whom you can create, delete and change databases manually on the localhost:5984/_utils/# <br />
```
Comment (muralikrishna8) [inline README.md]:
```suggestion
3. Create an admin user in couchdb with whom you can create, delete and change databases manually on the [localhost:5984/_utils/#](http://localhost:5984/_utils/#) <br />
```

File: README.md [hunk 7]
```diff
@@ -1,29 +1,55 @@
...
+If your admin user in couchdb is not with the name 'admin' and password 'admin', change this data in the file \" instal.config \" from this very project.
+4. Install couchdb lucene 2.0
+5. Install gulp
```
Comment (muralikrishna8) [inline README.md]:
```suggestion
5. Install gulp `npm install gulp -g`
```

File: oneClickInstaller.sh (hunk 1 above)
Comment (diangarkov) [reply oneClickInstaller.sh]: I forgot to add it as a file  - I added it and committed it, can you see it now or should I do a new pull request?

File: checkPreRequisites.sh [hunk 8]
```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+echo '-------------------------------------------'
+nodeVersion=$(node --version)
+
+if [[ nodeVersion > 4.0 ]] || [[ nodeVersion == 4.0 ]]; then
```
Comment (muralikrishna8) [inline checkPreRequisites.sh]: Please update the minimum version to **8.0**

File: oneClickInstaller.sh (hunk 1 above)
Comment (muralikrishna8) [reply oneClickInstaller.sh]: Yes I can see it now. 👍

File: README.md (hunk 5 above)
Comment (diangarkov) [reply README.md]: ok I do it

File: checkPreRequisites.sh (hunk 8 above)
Comment (diangarkov) [reply checkPreRequisites.sh]: done

File: README.md (hunk 7 above)
Comment (diangarkov) [reply README.md]: improved

File: README.md (hunk 6 above)
Comment (diangarkov) [reply README.md]: improved

**Pull request merged**